lib/vsc/filesystem/oceanstor.py

```diff
@@ -145,32 +145,41 @@ def __init__(self, *args, **kwargs):
         self.client = OceanStorClient(*args, **kwargs)
 
 
+class OceanStorOperationError(PosixOperationError):
+    pass
+
+
 class OceanStorOperations(with_metaclass(Singleton, PosixOperations)):
-    def __init__(self, url, username=None, password=None):
-        """Initialize REST client and request authentication token"""
+    def __init__(self, api_url, nfs_ips, username, password):
+        """
+        Initialize REST client and request authentication token
+
+        @type api_url: string with URL of REST API, only scheme and FQDM of server is needed
+        @type nfs_ips: list of IPs or FQDM of the NFS servers (if string: 1 IP)
```

Comment:
not for this PR but we should be able to query that from the API

Reply:
good idea, I found the interface for the NFS shares in the REST API. I'll add it in the next version.